cache.rst

```diff
@@ -415,23 +407,13 @@ case the value needs to be recalculated.
 
             <framework:config>
                 <framework:cache>
-                    <framework:pool name="my_cache_pool" adapter="cache.adapter.psr6" provider="app.my_cache_chain_adapter"/>
-                    <framework:pool name="cache.my_redis" adapter="cache.adapter.redis" provider="redis://user:password@example.com"/>
-                    <framework:pool name="cache.apcu" adapter="cache.adapter.apcu"/>
-                    <framework:pool name="cache.array" adapter="cache.adapter.array"/>
+                    <framework:pool name="my_cache_pool" default-lifetime="31536000">
+                        <framework:adapter name="cache.adapter.array" />
+                        <framework:adapter name="cache.adapter.apcu" />
+                        <framework:adapter name="cache.adapter.redis" provider="app.cache.provider.redis" />
```

Comment:
the provider should be `redis://user:password@example.com` if we want to match the yaml definition